www/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
I was about to merge this, but it seems like your editor has added the BOM at the start of the file.
I'd prefer to have consistent code (BOM nowhere or BOM everywhere) until we decide how to handle BOM : could you remove it?